src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -327,6 +327,71 @@ export default defineConfig({
 });
 ```
 
+### Local development features
+
+When running `astro dev`, the adapter enables several Netlify platform features to ensure the environment matches production as closely as possible. These include:
+
+- A local [Netlify Image CDN](https://docs.netlify.com/build/image-cdn/overview/) server. This is used for [images](#netlify-image-cdn-support) by default.
+- A local [Netlify Blobs](https://docs.netlify.com/build/data-and-storage/netlify-blobs/) server. This is used for [sessions](#sessions) by default
+- [Redirects, rewrites](https://docs.netlify.com/manage/routing/redirects/overview/) and [headers](https://docs.netlify.com/manage/routing/headers/) from your Netlify config
+- Access to [Netlify Edge Context](/en/guides/integrations-guide/netlify/#accessing-edge-context-from-your-site) in on-demand pages
```

Comment:
```suggestion
- Access to [Netlify Edge Context](#accessing-edge-context-from-your-site) in on-demand pages
```
Doesn't make a huge difference, but does make it easier for editors of the docs to know that this is a link within the same page.